2015/06/05 22:52:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/prueba/user
---
2015/06/05 23:08:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/prueba/user
---
2015/06/05 23:08:37 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "user".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('user')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/prueba/user
---
2015/07/18 17:29:46 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "postulante".' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('postulante')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('postulante')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/postulante...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#4 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/prueba/site/postulante/index
---
2015/07/18 17:31:35 [error] [php] include(USER.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:432)
Stack trace:
#0 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(29): spl_autoload_call()
#1 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/login
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (29)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 17:32:46 [error] [php] include(USER.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:432)
Stack trace:
#0 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(29): spl_autoload_call()
#1 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/login
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (29)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 17:35:30 [error] [php] include(USER.php): failed to open stream: No such file or directory (C:\xampp\htdocs\yii\framework\YiiBase.php:432)
Stack trace:
#0 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(29): spl_autoload_call()
#1 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#2 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute()
#4 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#5 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): LoginForm->validate()
#6 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#13 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/login
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (29)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:04:11 [error] [system.db.CDbCommand] CDbCommand::() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '='21645323' ANDContrasena='Holawe' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM PostulanteWHERE Cedula='21645323' ANDContrasena='Holawe.
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (26)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:04:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '='21645323' ANDContrasena='Holawe' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM PostulanteWHERE Cedula='21645323' ANDContrasena='Holawe' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbCommand.php(377): CDbCommand->queryInternal('', 0, Array)
#1 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(26): CDbCommand->query()
#2 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:05:50 [error] [system.db.CDbCommand] CDbCommand::() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '='21645323' ANDContrasena='gfg' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM PostulanteWHERE Cedula='21645323' ANDContrasena='gfg.
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (26)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:05:50 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '='21645323' ANDContrasena='gfg' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM PostulanteWHERE Cedula='21645323' ANDContrasena='gfg' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbCommand.php(377): CDbCommand->queryInternal('', 0, Array)
#1 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(26): CDbCommand->query()
#2 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:07:40 [error] [system.db.CDbCommand] CDbCommand::() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ANDContrasena='Holawe'' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM Postulante WHERE Cedula='21645323' ANDContrasena='Holawe'.
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (26)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:07:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ANDContrasena='Holawe'' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM Postulante WHERE Cedula='21645323' ANDContrasena='Holawe'' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbCommand.php(377): CDbCommand->queryInternal('', 0, Array)
#1 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(26): CDbCommand->query()
#2 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:08:36 [error] [system.db.CDbCommand] CDbCommand::() failed: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ANDContrasena='Holawe'' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM postulante WHERE Cedula='21645323' ANDContrasena='Holawe'.
in C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php (26)
in C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (94)
2015/07/18 18:08:36 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand failed to execute the SQL statement: SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ANDContrasena='Holawe'' at line 1. The SQL statement executed was: SELECT Cedula, Contrasena FROM postulante WHERE Cedula='21645323' ANDContrasena='Holawe'' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:543
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\CDbCommand.php(377): CDbCommand->queryInternal('', 0, Array)
#1 C:\xampp\htdocs\yii\prueba\protected\components\UserIdentity.php(26): CDbCommand->query()
#2 C:\xampp\htdocs\yii\prueba\protected\models\LoginForm.php(52): UserIdentity->authenticate()
#3 C:\xampp\htdocs\yii\framework\validators\CInlineValidator.php(42): LoginForm->authenticate('password', Array)
#4 C:\xampp\htdocs\yii\framework\validators\CValidator.php(201): CInlineValidator->validateAttribute(Object(LoginForm), 'password')
#5 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CValidator->validate(Object(LoginForm), NULL)
#6 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(94): CModel->validate()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('login')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/login')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#14 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/prueba/site/login
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:09:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "postulante/create".' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/crea...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#2 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/prueba/postulante/create
HTTP_REFERER=http://localhost:7777/yii/prueba/site/login
---
2015/07/18 18:21:27 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'You are not authorized to perform this action.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:182
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(145): CAccessControlFilter->accessDenied(Object(CWebUser), 'You are not aut...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('postulante/admi...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/yii/prueba/postulante/admin
HTTP_REFERER=http://localhost:7777/yii/prueba/postulante/index
---
2015/07/18 18:46:03 [error] [php] Undefined variable: model (C:\xampp\htdocs\yii\prueba\protected\views\site\login.php:28)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(32): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/index
in C:\xampp\htdocs\yii\prueba\protected\views\site\login.php (28)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (32)
in C:\xampp\htdocs\yii\prueba\index.php (13)
2015/07/18 18:46:06 [error] [php] Undefined variable: model (C:\xampp\htdocs\yii\prueba\protected\views\site\login.php:28)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php(32): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(184): CWebApplication->processRequest()
#10 C:\xampp\htdocs\yii\prueba\index.php(13): CWebApplication->run()
REQUEST_URI=/yii/prueba/site/index
in C:\xampp\htdocs\yii\prueba\protected\views\site\login.php (28)
in C:\xampp\htdocs\yii\prueba\protected\controllers\SiteController.php (32)
in C:\xampp\htdocs\yii\prueba\index.php (13)
